spec/spec_helper.rb

```diff
@@ -17,3 +17,7 @@
 Dir["spec/fake_app/**/*.rb"].each do |file|
   require File.join(File.dirname(__FILE__), "..", file)
 end
+
+RSpec.configure do |config|
```

Comment:
The concern is your tests are failing on Travis. Try running rake and make sure that works.